README.md

```diff
@@ -86,3 +86,23 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+
+Docker commands
+These are the commands to build the image and run docker
+For Dev instance
+docker build --target development --tag todo-app:dev .
```

Comment:
This is a markdown file (.md file extension). Single line breaks don't lead to line breaks in the formatted file:

https://github.com/rajeshdewan/-DevOps-Course-Starter/blob/40b44d3076c31498b3587fc5ac4944c157733d27/README.md